dev/package-examples/endpoint-0.0.1/elasticsearch/index-template/endpoint-index-template.json

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
We generate the index template in the package manager from the `fields.yml` files. This has the advantage that we can generate the template and the Kibana index pattern for it. And YAML allows to add a bit more comments.

Reply:
@ruflin @jen-huang I'm having some trouble finding the tool that handles parsing the fields.yml files. Could you point me to where that tool lives?

Reply:
Sonja mentioned these places:

currently, generation of index templates happens here https://github.com/elastic/kibana/blob/feature-ingest/x-pack/legacy/plugins/epm/server/lib/elasticsearch/template/install.ts , and of index patterns here: https://github.com/elastic/kibana/blob/feature-ingest/x-pack/legacy/plugins/epm/server/lib/kibana/index_pattern/install.ts